### PR TITLE
Updates to setup guides for APIs

### PR DESCRIPTION
**Pull request by nataliekwong**
## What
Goal is to improve the usability of docs for Cloud users by centering on setup within the app.

Creates new in-app docs for:
- Asana
- BambooHR
- Bing Ads
- SendGrid
- Shopify

Update existing docs with improved setup information (so non-Cloud users can benefit too):
- Shopify

Update existing docs with OSS tags
- Amplitude
- US Census
- Waiteraid
- Webflow
- Woocommerce

## 🚨 User Impact 🚨
*Are there any breaking changes? What is the end result perceived by the user?*
None

### CODE REVIEW

File: docs/integrations/sources/bing-ads.inapp.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+2. [Request user consent](https://docs.microsoft.com/en-us/advertising/guides/authentication-oauth-consent?view=bingads-13l) to get the authorization code.
+3. Use the authorization code to [get a refresh token](https://docs.microsoft.com/en-us/advertising/guides/authentication-oauth-get-tokens?view=bingads-13).
+
+:::note
+
+The refresh token expires every 90 days. Repeat the authorization process to get a new refresh token. The full authentication process described [here](https://docs.microsoft.com/en-us/advertising/guides/get-started?view=bingads-13#access-token).
+Please be sure to authenticate with the email (personal or work) that you used to sign in to the Bing ads/Microsoft ads platform.
+:::
+
+4. Request your [Microsoft developer token](https://docs.microsoft.com/en-us/advertising/guides/get-started?view=bingads-13#get-developer-token) in the Microsoft Advertising Developer Portal account tab.
```
Comment (sherifnada) [inline docs/integrations/sources/bing-ads.inapp.md:15]: fwiw this will go away soon https://github.com/airbytehq/airbyte/issues/26380

**Review by sherifnada — APPROVED** [review]

**Review by sherifnada — APPROVED** [review]

File: docs/integrations/sources/amplitude.md [hunk 2]
```diff
@@ -73,3 +73,4 @@ The Amplitude connector ideally should gracefully handle Amplitude API limitatio
 | 0.1.2   | 2021-09-21 | [6353](https://github.com/airbytehq/airbyte/pull/6353)   | Correct output schemas on cohorts, events, active\_users, and average\_session\_lengths streams           |
 | 0.1.1   | 2021-06-09 | [3973](https://github.com/airbytehq/airbyte/pull/3973)   | Add AIRBYTE\_ENTRYPOINT for kubernetes support                                                            |
 | 0.1.0   | 2021-06-08 | [3664](https://github.com/airbytehq/airbyte/pull/3664)   | New Source: Amplitude                                                                                     |
+<!-- /env:oss -->
```
[the remark this answers is not in the document]
Comment (nataliekwong) [reply docs/integrations/sources/amplitude.md:76]: Yep! That is what we will move toward as I understand from the Metadata plan.

File: docs/integrations/sources/bamboo-hr.inapp.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+## Prerequisites
+
+* BambooHR [API Key](https://documentation.bamboohr.com/docs#authentication)
+
+
+## Setup guide
+1. Name your BambooHR connector
```
[the remark this answers is not in the document]
Comment (nataliekwong) [reply docs/integrations/sources/bamboo-hr.inapp.md:7]: I ended up randomly on this one; I had started just exploring sources from top alphabetically (A - Z) and found this doc particularly bad.

File: docs/integrations/sources/bamboo-hr.inapp.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+2. Enter your `api_key`. To generate an API key, users should log in and click their name in the upper right-hand corner of any page to get to the user context menu. If they have sufficient permissions, there will be an "API Keys" option in that menu to go to the page.
```
[the remark this answers is not in the document]
Comment (nataliekwong) [reply docs/integrations/sources/bamboo-hr.inapp.md:8]: Good point; I took this from their API docs. Adjusting to administrator.

File: docs/integrations/sources/bing-ads.inapp.md (hunk 1 above)
Comment (nataliekwong) [reply docs/integrations/sources/bing-ads.inapp.md:15]: Glad to hear it! This process looks painful.

**Pull request merged**